modules/core/src/main/java/org/apache/ignite/internal/processors/rest/GridRestProcessor.java

```diff
@@ -1039,6 +1043,30 @@ private void startProtocol(GridRestProtocol proto) throws IgniteCheckedException
         X.println(">>>   handlersSize: " + handlers.size());
     }
 
+    /**
+     * Logs REST request processing result.
+     *
+     * @param req Received REST request.
+     * @param fut Result of REST request processing.
+     */
+    private void logRestRequestResult(GridRestRequest req, IgniteInternalFuture<GridRestResponse> fut) {
+        if (log == null || !log.isDebugEnabled())
```

Comment:
When log may be null?

Reply:
Seems, we can do not register listener in case debug disabled.

Reply:
My mistake. Ignite log4j logger is used if no specified. Done.